core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -215,6 +219,28 @@ export class ContributionOpportunitiesBackendApiService {
       return [];
     }
   }
+
+  async saveUserLanguagePreferenceAsync(languageCode: string):
+  Promise<Object> {
```

Comment:
I think we can return a void Promise.

Reply:
I think we cannot. Kindly refer to my previous comment.
Comment link: https://github.com/oppia/oppia/pull/15460#discussion_r876249148

Reply:
Thank you for the reference. It was a silly mistake on my end. I have replaced this with `<void>`.